Javascript/reset.js

```diff
@@ -1,625 +1,650 @@
-        }
-
-        if (player.shopUpgrades.instantChallengeBought && player.currentChallengeRein == ""){
-            player.challengecompletions.one = player.highestchallengecompletions.one
-            player.challengecompletions.two = player.highestchallengecompletions.two
-            player.challengecompletions.three = player.highestchallengecompletions.three
-            player.challengecompletions.four = player.highestchallengecompletions.four
-            player.challengecompletions.five = player.highestchallengecompletions.five
-
-        }
-
-        player.reincarnatenocoinupgrades = true;
-        player.reincarnatenocoinorprestigeupgrades = true;
-        player.reincarnatenocoinprestigeortranscendupgrades = true;
-        player.reincarnatenocoinprestigetranscendorgeneratorupgrades = true;
-        player.reincarnatenoaccelerator = true;
-        player.reincarnatenomultiplier = true;
-
-        if (player.reincarnationcounter < player.fastestreincarnate && player.currentChallengeRein == "") {
-            player.fastestreincarnate = player.reincarnationcounter;
-        }
-        player.reincarnationcounter = 0;
-
-
-        if (player.autoResearchToggle && player.autoResearch > 0.5){
-            buyResearch(player.autoResearch, true)
-        }
-        calculateRuneLevels();
-        calculateAnts();
-    }
-
-
-
-    if (i > 0.5) {
-        player.unlocks.prestige = true
-    }
-    if (i > 1.5) {
-        player.unlocks.transcend = true
-    }
-    if (i > 2.5) {
-        player.unlocks.reincarnate = true
-    }
-    if (!fast){
-    revealStuff();
+    player.transcendcounter = 0;
+
+
+
+  }
+  if (i > 2.5) {
+
+    player.researchPoints += Math.floor(obtainiumGain);
+
+    let opscheck = obtainiumGain / (1 + player.reincarnationcounter)
+    if (opscheck > player.obtainiumpersecond) {
+      player.obtainiumpersecond = opscheck
     }
+
+    resetUpgrades(3, fast);
+    player.coinsThisReincarnation = new Decimal("100");
+    player.firstOwnedMythos = 0;
+    player.firstCostMythos = new Decimal("1");
+    player.secondOwnedMythos = 0;
+    player.secondCostMythos = new Decimal("1e2");
+    player.thirdOwnedMythos = 0;
+    player.thirdCostMythos = new Decimal("1e4");
+    player.fourthOwnedMythos = 0;
+    player.fourthCostMythos = new Decimal("1e8");
+    player.fifthOwnedMythos = 0;
+    player.fifthCostMythos = new Decimal("1e16");
+    player.firstGeneratedParticles = new Decimal("0");
+    player.secondGeneratedParticles = new Decimal("0");
+    player.thirdGeneratedParticles = new Decimal("0");
+    player.fourthGeneratedParticles = new Decimal("0");
+    player.fifthGeneratedParticles = new Decimal("0");
+
+    player.reincarnationCount += 1;
+
+    player.transcendPoints = new Decimal("0");
+    player.reincarnationPoints = player.reincarnationPoints.add(reincarnationPointGain);
+    player.reincarnationShards = new Decimal("0");
+    player.challengecompletions = {
+      one: 0,

```

Comment:
Why are these not all the same (using player.challengecompletions.one)? It's been used elsewhere

Reply:
I failed to read that, can you give more details please?